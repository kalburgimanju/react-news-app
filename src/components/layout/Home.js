import PropTypes from "prop-types";
import React, { Component } from "react";

import ArticleListContainer from "../../containers/ArticleListContainer";

const propTypes = {};

const defaultProps = {};

class Home extends Component {
  render() {
    return (
      <section>
        <div className="wrapper">
          <ArticleListContainer />
        </div>
      </section>
    );
  }
}

Home.propTypes = propTypes;
Home.defaultProps = defaultProps;

export default Home;
