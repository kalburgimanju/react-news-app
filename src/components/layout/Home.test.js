import React from "react";
import Home from "./Home";

it("render Home", () => {
  const wrapper = shallow(<Home />);
  expect(wrapper).toMatchSnapshot();
});
