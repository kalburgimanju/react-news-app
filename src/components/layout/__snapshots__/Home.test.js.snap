// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Home 1`] = `
<section>
  <div
    className="wrapper"
  >
    <Connect(ArticleListContainer) />
  </div>
</section>
`;
